src/mpid/ch4/src/ch4_probe.h

```diff
@@ -31,6 +32,7 @@ MPL_STATIC_INLINE_PREFIX int MPID_Probe(int source,
         goto fn_exit;
     }
 
+    av = MPIDIU_comm_rank_to_av(comm, source);
```

Comment:
Should this move down to the `#else` block so we don't do an unnecessary lookup if using non-exclusive shared memory mode?